docs/docsite/rst/dev_guide/testing/sanity/future-import-boilerplate.rst

```diff
@@ -0,0 +1,51 @@
+Sanity Tests » from __future__ boilerplate
+==========================================
+
+Most Python files should include the following boilerplate at the top of the file, right after the
+comment header:
+
+.. code-block:: python
+
+    from __future__ import (absolute_import, division, print_function)
+
+This uses Python3 semantics for absolute vs relative imports, division, and print.  By doing this,
```

Comment:
Docs should refer to `Python 2` and `Python 3` instead of `Python2` and `Python3`.

Reply:
Fixed